src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <h1> FORM </h1>
+      <p> Будь ласка заповність форму</p>
+      <script type="text/javascript" src="./main.js"></script>
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+     Surname:
+          <input type="text" name="username"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+        Name:
+          <input type="text" name="username"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+        How old are You?
+          <input type="ages" name="ages"
+            minlength="1"
+            maxlength="100"
+            pattern="\d+"
+            required
+           >
+        </div>
+
+        <div class="form-field">
+        Full data of birth:
+          <input type="date" name="date"
+            required
+          >
+        </div>
+
+        <label class="form-field">
+          <input type="checkbox" name="terms"
+            required
+            >
+        </label>
+
+        <label>I accept the term of the agreement</label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-field">
+          Email:
+          <input type="email" name="Email"
+            required
+          >
+        </div>
+        <div class="form-field">
```

Comment:
you have to add an empty line between similar children
```suggestion
        </div>
        
        <div class="form-field">
```